### PR TITLE
Create extra-list.yaml

### PR DESCRIPTION
**Pull request by muhammad-levi**
Add ability to deploy extra manifests via the Helm chart (managed in the same file of values)

### CODE REVIEW

**Comment by jkiddo** [comment]
@chgl can you have a look?

File: charts/hapi-fhir-jpaserver/templates/extra-list.yaml
Comment (chgl) [inline charts/hapi-fhir-jpaserver/templates/extra-list.yaml:1]: This common library chart comes from bitnami, but isn't referenced in this chart so it won't work.

Comment (chgl) [inline charts/hapi-fhir-jpaserver/templates/extra-list.yaml:1]: extraDeploy should be included in the values.yaml and the readme re-generated to update it

**Review by chgl — CHANGES_REQUESTED** [review]

**Comment by jkiddo** [comment]
Closed as deemed stale

**Pull request closed without merging**